test/tokens/_api.scss

```diff
@@ -131,8 +131,27 @@
           'ERROR [lookup-alias] Alias "that->baz" not found in map'
         )
       );
+
+      config.$handle-missing-keys: null;
+    }
+
+    @include it('Handles CSS values in maps') {
+      $gradients: (
+        'accent':
+          linear-gradient(hsl(31deg 94% 52%) 10%, hsl(274deg 95% 57%) 80%),
+      );
```

Comment:
lets clarify this by using hex notation for the colors, and updating the test description so it explains the issue a bit more?

Reply:
Sounds good - will do.